contracts/core/liquidators/impl/TwoAssetPriceBoundedLinearAuction.sol

```diff
@@ -92,6 +102,72 @@ contract TwoAssetPriceBoundedLinearAuction is LinearAuction {
         );
     }
 
+    /**
+     * Calculates the minimumBid. First calculates the minimum token flow for the pair at fair value using
+     * maximum natural unit of two Sets. If that token flow is below 1000 units then calculate minimumBid
+     * as such:
+     *
+     * minimumBid = maxNaturalUnit*1000/min(tokenFlow)
+     *
+     * Else, set minimumBid equal to maxNaturalUnit. This is to ensure that around fair value there is ample
+     * granualarity in asset pair price changes and not large discontinuities.
+     *
+     * @param _auction            Auction object
+     * @param _currentSet         CurrentSet, unused in this implementation
+     * @param _nextSet            NextSet, unused in this implementation
+     */
+    function calculateMinimumBid(
+        Auction.Setup storage _auction,
+        ISetToken _currentSet,
+        ISetToken _nextSet
+    )
+        internal
+        view
+        returns (uint256)
+    {
+        // Get full Unit amount and price for each asset
+        AssetInfo memory assetOne = getAssetInfo(_auction.combinedTokenArray[0]);
+        AssetInfo memory assetTwo = getAssetInfo(_auction.combinedTokenArray[1]);
+
+        // Calculate current spot price as assetOne/assetTwo
+        uint256 spotPrice = calculateSpotPrice(assetOne.price, assetTwo.price);
+
+        // Calculate auction price at current asset pair spot price
+        uint256 auctionFairValue = convertAssetPairPriceToAuctionPrice(
+            _auction,
+            spotPrice,
+            assetOne.fullUnit,
+            assetTwo.fullUnit
+        );
+
+        uint256 minimumBidMultiplier = 0;
+        for (uint8 i = 0; i < _auction.combinedTokenArray.length; i++) {
+            // Get token flow at fair value for asset i
+            (
+                uint256 tokenInflowScaled,
+                uint256 tokenOutflowScaled
+            ) = Auction.calculateInflowOutflow(
+                _auction.combinedCurrentSetUnits[i],
+                _auction.combinedNextSetUnits[i],
+                10 ** 18,
+                auctionFairValue
+            );
+
+            // One returned number from previous function will be zero so use max to get tokenFlow
+            uint256 tokenFlowScaled = Math.max(tokenInflowScaled, tokenOutflowScaled);
+
+            // Divide minimum spot token flow (1000 units) by token flow if more than minimumBidMultiplier
+            // update minimumBidMultiplier
+            minimumBidMultiplier = MIN_SPOT_TOKEN_FLOW_SCALED.div(tokenFlowScaled) > minimumBidMultiplier ?
```

Comment:
If you still have variable space, can put `MIN_SPOT_TOKEN_FLOW_SCALED.div(tokenFlowScaled)` into a var